vars/runContinuousIntegrationPipeline.groovy

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env groovy
+
+/****************************************************************************
+ * vars/runContinuousIntegrationPipeline.groovy 
+ * Logic for the Continuous Integration pipeline
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+def call() {
+  pipeline {
+    agent {
+      node {
+	label 'ubuntu'
+      }
+    }
+    environment {
+      EXAMPLE_ENV = 'example_value'
+      // GIT_CREDENTIALS_ID = '<uuid>'
+      NUTTX_APPS_REPO_URL = 'https://github.com/apache/incubator-nuttx-apps'
+      NUTTX_TESTING_REPO_URL = 'https://github.com/apache/incubator-nuttx-testing'
+    }
+    stages {
+      stage('Checkout') {
+	steps {
+	  deleteDir() // clean up our workspace
```

Comment:
@maht, I used the scripts setup jenkins pipiline job locally well. But I notice the nuttx repo firstly checkouted before deleteDir from the build log. Is this right?

Reply:
No, sorry, this is not ok. It need to update the script. It should be checked out in the `nuttx/` path. Let me fix it.